pmg/api/v1.py

```diff
@@ -315,6 +315,7 @@ def committee_list():
 @api.route('/<string:resource>/', )
 @api.route('/<string:resource>/<int:resource_id>/', )
 @load_user()
+@cache.memoize(unless=lambda: no_cache(request))
```

Comment:
I'd suggest renaming `no_cache` to `should_skip_cache` or similar. `no_cache` sounds like it should be used to prevent things from being cached.

Reply:
ah right - sure